app/views/charge.njk

```diff
@@ -63,60 +63,63 @@
               </div>
             </div>
             {% if allowApplePay or allowGooglePay %}
-              <h1 class="govuk-heading-l">{{ __p("cardDetails.enterPaymentDetails") }}</h1>
+              <h1 class="govuk-heading-l web-payment-button-heading">{{ __p("cardDetails.enterPaymentDetails") }}</h1>
```

Comment:
This heading needs to be hidden when there are no Apple/Google Pay buttons.
So add the `web-payment-button-heading` class to it.